pkg/scheduler/eventhandlers.go

```diff
@@ -255,11 +255,11 @@ const (
 )
 
 // WaitForHandlersSync waits for EventHandlers to sync.
-// It returns true if it was successful, false  if the controller should shut down
+// It returns true if it was successful, false if the controller should shut down
 func (sched *Scheduler) WaitForHandlersSync(ctx context.Context) bool {
```

Comment:
obtain the log from the context here. We shouldn't be querying the context in each iteration.

Reply:
Like this?
```
func (sched *Scheduler) WaitForHandlersSync(ctx context.Context) bool {
	logger := klog.FromContext(ctx)
```

Reply:
It's already updated